### PR TITLE
Adding silent as an input

### PR DESCRIPTION
**Pull request by shigupt202**

### CODE REVIEW

File: src/run.ts [hunk 1]
```diff
@@ -138,8 +142,12 @@ class KustomizeRenderEngine extends RenderEngine {
             throw Error(util.format("kustomizationPath %s does not exist. Please check whether file exists or not.", kustomizationPath));
         }
         
+        var isSilent = true;
+        var silentInput = core.getInput('silent', {required : false});
+        if(silentInput && silentInput == 'false')
```
Comment (ds-ms) [inline src/run.ts]: NIT: indent

File: src/run.ts [hunk 2]
```diff
@@ -138,8 +142,12 @@ class KustomizeRenderEngine extends RenderEngine {
...
+            isSilent = false;
         const options = {
-            silent: true
+            silent: isSilent
```
Comment (ds-ms) [inline src/run.ts]: Just use !!silent instead

Comment (ds-ms) [reply src/run.ts]: And get rid of the above step

File: src/run.ts [hunk 3]
```diff
@@ -14,10 +14,10 @@ import { getKomposePath } from "./kompose-util"
```
Comment (ds-ms) [inline src/run.ts]: You missed commiting the compiled file

Comment (shigupt202) [reply src/run.ts]: Added the compiled file

**Pull request merged**